### PR TITLE
added *.pb.go generation instructions to README

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,6 @@ repository. This repository *will not* depend on any other repos
 
 All components of an Istio installation operate on a shared vocabulary of attributes,
 as defined in this repo.
+
+## Updating
+After [protobuf](https://github.com/google/protobuf) definitions are updated, corresponding`*pb.go`files must be generated by running `bazel build ...`. The files are generated in `bazel-genfiles` directory, in a subdirectory matching the directory of the defintion, e.g. `bazel-genfiles/proxy/v1/config`. The generated files must be copied to the directory of the updated profobuf defintions and committed together with the defintion changes.
```
Comment (ayj) [inline README.md]: You can also run https://github.com/istio/api/blob/master/scripts/generate-protos.sh which will compile protos and copy over *.pb.go to target directory alongside *.proto files.

**Comment by vadimeisenbergibm** [comment]
@ayj @rshriram updated the README to use scripts/generate-protos.sh

**Comment by ZackButcher** [comment]
Do we have bots wired up for istio/api? If not, I can merge manually.

**Pull request merged**